Quintana_Jerrod/Algorithms/10_12_16.html

```diff
@@ -0,0 +1,67 @@
+<html>
+<head>
+  <title></title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  <script type="text/javascript">
+  $(docuemnt).ready(function(){
+    SLL.prototype.length = function(){
+      if (this.head == null) {return 'list is empty'}
+      var count = 0;
+      var current = this.head;
+      while (current) {
+        count ++;
+        current = current.next;
+      }
+      return count;
+    }
+
+    SLL.prototype.max= function(){
+      if (this.head == null) { return 'list is empyt'}
+      var max = this.head.val;
+      var current = this.head;
+      while (current){
+        if (current.val > max)
+          max = current.val;
+        current = current.next;
+      }
+      return max;
+    }
+
+    SLL.prototype.min= function(){
+      if (this.head == null) { return 'list is empyt'}
+      var min = this.head.val;
+      var current = this.head;
+      while (current){
+        if (current.val < min)
+          min = current.val;
+        current = current.next;
+      }
+      return min;
+    }
+
+    SLL.prototype.avg= function(){
+      if (this.head == null) { return 'list is empyt'}
+      var avg = 0;
+      var length = 0;
+      var current = this.head;
+      while (current){
+        count ++;
+        avg += current.val;
+        current = current.next;
+      }
+      avg = avg/length;
+      return avg;
+    }
+
```

Comment:
Where are the tests? Call your functions with some sample data to see if they work properly